runtime/plugin/rplugin.vim

```diff
@@ -29,22 +23,32 @@ function! s:GetManifestPath() abort
 endfunction
 
 " Old manifest file based on known script locations.
-function! s:GetOldManifestPath() abort
+function! s:GetOldManifestPaths() abort
   let prefix = exists('$MYVIMRC')
         \ ? $MYVIMRC
         \ : matchstr(get(split(execute('scriptnames'), '\n'), 0, ''), '\f\+$')
-  return fnamemodify(expand(prefix, 1), ':h')
+  let origpath = fnamemodify(expand(prefix, 1), ':h')
         \.'/.'.fnamemodify(prefix, ':t').'-rplugin~'
+  if !has('win32')
+    return [origpath]
+  endif
+  " Windows used to use $APPLOCALDATA/nvim but stdpath('data') is
+  " $XDG_DATA_DIR/nvim-data
+  let pseudostdpath = exists('$LOCALAPPDATA') ? '$LOCALAPPDATA' : '~/AppData/Local'
+  let pseudostdpath = fnamemodify(expand(pseudostdpath), ':p')
+  return [substitute(pseudostdpath, '[/\\]\=$', '/', '') . 'nvim', origpath]
 endfunction
 
 function! s:GetManifest() abort
   let manifest = s:GetManifestPath()
   if !filereadable(manifest)
     " Check if an old manifest file exists and move it to the new location.
-    let old_manifest = s:GetOldManifestPath()
-    if filereadable(old_manifest)
-      call rename(old_manifest, manifest)
-    endif
+    for old_manifest in s:GetOldManifestPaths()
```

Comment:
Currently this tries:

```
C:\Users\foo\AppData\Local/nvim
C:\Users\foo\AppData\Local\nvim/.init.vim-rplugin~
```

but I think we want the first one to be:

```
C:\Users\foo\AppData\Local/nvim/rplugin.vim
```

Reply:
This fixes it:

      return [substitute(pseudostdpath, '[/\\]\=$', '/', '') . 'nvim/rplugin.vim', origpath]

Reply:
Fixed.